src/SimpleAccounting/Model/ProjectData.cs

```diff
@@ -227,37 +227,13 @@ public void SelectYear(string yearName)
             this.YearChanged(this, EventArgs.Empty);
         }
 
-        public void ShowAddBookingDialog(bool showInactiveAccounts)
+        private EditBookingViewModel BookingFromJournal(AccountingDataJournalBooking journalEntry, bool isEdit, bool showInactiveAccounts)
```

Comment:
`private` methods should be placed after `public` (and `protected`) methods.
If you have "ReSharper" or "Rider" please try the cleanup profile "lg2".